package org.revature.p1.utils.exceptions;

public class DuplicateUserException extends RuntimeException {
}
